### ISSUE TITLE
Maven Plug-in

### ISSUE THREAD

**Issue by GoogleCodeExporter** [mernst/randoop#40]
```
I have a plug-in already working but want to add more stuff to it: 
https://bitbucket.org/javydreamercsw/randoop-maven-plugin

I have some questions/issues:
1) I would need Randoop in Maven repository. I can do it myself but I guess you 
would like to have control over that.
2) I would like to add some of your guys to have to my bitbucket repo so they 
can contribute.
3) Do you have a place to host the Maven site that I'll generate for it?
```

Original issue reported on code.google.com by `javier.o...@gmail.com` on 29 Nov 2012 at 3:47

**Comment by mernst** [comment]
Thanks!  I have added a link to this Maven plug-in from the Randoop manual.

**Comment by zaplatynski** [comment]
Hey, what is the status of the Maven plugin? Seems not active any more... I would like to contribute if it possible or rewite one since I have some experience in writing Maven plugins.

**Comment by bjkeller** [comment]
I don't know anything about it.  It seems like it would need to be resurrected or replaced.

I added an empty repo https://github.com/randoop/randoop-maven-plugin. You could build your plugin and then do a PR to that repo.

**Comment by zaplatynski** [comment]
Ok, thanks. Is there any repository available from where i can create a Maven dependency?

**Comment by bjkeller** [comment]
Sorry, I don't know of one.

**Comment by zaplatynski** [comment]
Never mind. I try something. I will open a new a new issue if I have new issues to report.

**Comment by jeacott1** [comment]
@zaplatynski did you ever get to building a new maven plugin?

**Comment by zaplatynski** [comment]
Yes, I am waiting for the promised Maven dependency by Ben Keller since
then because I made an awful hack to get the randdoop via command line to
work.
I did no want to create an pull request for that and waited since then.
Now someone just deleted the upstream repository so I can not create any
pull request.

The plugin with the awful hack is still on my profile available:
https://github.com/zaplatynski/randoop-maven-plugin

Regards,
Marian

2018-03-06 3:06 GMT+01:00 jeacott1 <notifications@github.com>:
